src/blocks/blocks.controller.ts

```diff
@@ -62,16 +62,9 @@ export class BlocksController {
       }),
     )
     upsertBlocksDto: UpsertBlocksDto,
-  ): Promise<List<SerializedBlock>> {
-    const blocks = await this.blocksTransactionsLoader.bulkUpsert(
-      upsertBlocksDto,
-    );
-    return {
-      object: 'list',
-      data: blocks.map((block) =>
-        serializedBlockFromRecordWithTransactions(block),
-      ),
-    };
+  ): Promise<void> {
+    await this.blocksTransactionsLoader.bulkUpsert(upsertBlocksDto);
+    return;
```

Comment:
What do you think about leaving the response like before? I have found it useful before for local development and don't think we'd lose anything.